src/backend.jl

```diff
@@ -36,6 +38,9 @@ function conda_cmd(args=``; io::IO=stderr)
     b = backend()
     if b == :MicroMamba
         MicroMamba.cmd(args, io=io)
+    elseif b == :Null
+        @error "Backend is 'Null' - no action taken"
+        return nothing
```

Comment:
I think just raise an error here. It's an internal function which should never be called with this backend anyway.